Documentation/gettingstarted/k8s-install-azure.rst

```diff
@@ -26,7 +21,47 @@ Setup a Kubernetes cluster on Azure. You can use any method available as long
 as your Kubernetes cluster has CNI enabled in the kubelet configuration. For
 simplicity of this guide, we will set up a managed AKS cluster:
 
-.. include:: k8s-install-aks-create-cluster.rst
+Prerequisites
+-------------
+
+Ensure that you have the `Azure Cloud CLI
+<https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_
+installed and log in to Azure with:
+
+.. code:: bash
+
+    az login
+
+To verify, confirm that the following command displays the set of available
+Kubernetes versions.
+
+.. code:: bash
+
+   az aks get-versions -l westus -o table
+
+Deploy the Cluster
+------------------
+
+.. note:: **Do NOT specify the '--network-policy' flag** when creating the
+    cluster, as this will cause the Azure CNI plugin to push down unwanted
+    iptables rules.
+
+.. code:: bash
+
+   export RESOURCE_GROUP_NAME=aks-test
+   export CLUSTER_NAME=aks-test
+   export LOCATION=westeurope
```

Comment:
Tiny nit: the previous command uses location `westus` but this command uses `westeurope`.